dragonchain/webserver/lib/dragonnet.py

```diff
@@ -80,13 +87,51 @@ def process_receipt_v1(block_dto: Dict[str, Any]) -> None:
                 _log.exception("matchmaking add_receipt failed!")
             # Update the broadcast system about this receipt
             broadcast_functions.set_receieved_verification_for_block_from_chain_sync(l1_block_id, level_received_from, block_model.dc_id)
+            # Send verifications to any configured urls
+            send_verification_notifications(level_received_from, block_model)
         else:
             _log.warning(
                 f"Chain {block_model.dc_id} (level {level_received_from}) returned a receipt that wasn't expected (possibly expired?) for block {l1_block_id}. Rejecting receipt"  # noqa: B950
             )
             raise exceptions.NotAcceptingVerifications(f"Not accepting verifications for block {l1_block_id} from {block_model.dc_id}")
 
 
+def get_notification_urls(key: str) -> set:
+    urls = None
+    try:
+        urls = VERIFICATION_NOTIFICATION[key]
+    except KeyError:
+        pass
+    if urls is None:
+        urls = []
+    return set(urls)
+
+
+def sign(message: str) -> Tuple[str, Optional[str]]:
+    dc_keys = keys.DCKeys()
+    signature = dc_keys.make_signature(message.encode("utf8"), crypto.SupportedHashes.sha256)
+    return (signature, dc_keys.pub)
```

Comment:
`dc_keys.pub` is an internal C secp256k1 struct. The chain's own public id is it's base58 encoded public key. I would recommend we use that instead.

It also seems unnecessary to return this for every signature, as a chain's public key won't change while it's running. Whatever function needing to use the public key string can call `keys.get_public_id()` when it needs it rather than passing this static key through a call stack every time